WordPressUI.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name          = "WordPressUI"
-  s.version       = "1.3.1"
+  s.version       = "1.3.2"
   s.summary       = "Home of reusable WordPress UI components."
 
   s.description   = <<-DESC
```

Comment:
If we are removing the `.xcassets` file we should remove the reference from the podspec too.

Reply:
Good call – done